### PR TITLE
feat: collections video grid

### PR DESCRIPTION
**Pull request by DanChae03**
# Description

Complete refactor of the videosgrid component. Create gridItem component that acts as the tiled element in the videosgrid. These components will be used in the collections page.

- Link to Basecamp Todo

# How should this PR be QA Tested?

- [x] Grid title should correctly render
- [x] Video duration should properly render (minutes and seconds)
- [x] Each video should have correct URL associated with it
- [x] Grid should render all videos passed in as prop array
- [x] Collections of Collections (See LUMO collection) should not render duration and should display episode count
- [x] Hover zoom animation works for each card.

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged into main

### CODE REVIEW

File: apps/watch/src/components/VideosGrid/VideosGrid.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+const Template: Story = ({ ...args }) => {
+  return <VideosGrid {...args} />
+}
+
+export const Default = Template.bind({})
+
+Default.args = {
+  videos
```
Comment (csiyang) [inline apps/watch/src/components/VideosGrid/VideosGrid.stories.tsx:19]: @DanChae03 This is rendering too many videos, most of it is not getting loaded in the story. We should decrease this to max 9 videos

**Review by csiyang — CHANGES_REQUESTED** [review]

**Comment by mikeallisonJS** [comment]
image
There is a strange hover state on the lower text

**Review by mikeallisonJS — DISMISSED** [review]

File: apps/watch/src/components/VideosGrid/VideoGridCard/VideoGridCard.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                    bottom: 5,
+                    right: 5,
+                    bgcolor: 'rgba(0, 0, 0, 0.5)',
+                    color: 'primary.contrastText',
+                    borderRadius: '8px',
+                    padding: '8px'
+                  }}
+                >
+                  <PlayArrow sx={{ fontSize: '1rem' }} />
+                  <Typography variant="body1" sx={{ lineHeight: '1rem' }}>
```
[the remark this answers is not in the document]
Comment (DanChae03) [reply apps/watch/src/components/VideosGrid/VideoGridCard/VideoGridCard.tsx:85]: Yeah it's not necessary, think it was just left over from the past

**Review by csiyang — CHANGES_REQUESTED** [review]

File: apps/watch/src/components/VideosGrid/VideoGridCard/VideoGridCard.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  return (
+    <Card
+      sx={{
+        boxShadow: 0,
+        borderRadius: '8px'
+      }}
+    >
+      {video == null ? (
+        <>
+          <CardMedia
```
[the remark this answers is not in the document]
Comment (csiyang) [reply apps/watch/src/components/VideosGrid/VideoGridCard/VideoGridCard.tsx]: After checking with @stevendiller the current darken behaviour is fine

**Review by tataihono — CHANGES_REQUESTED** [review]

File: apps/watch/src/components/VideoContainerPage/ContainerHero/ContainerHero.tsx [hunk 4]
```diff
@@ -7,19 +7,18 @@ import { ReactElement } from 'react'
...
+interface ContainerHeroProps {
   openDialog: () => void
 }
 
-export function ContainerHero({ video, openDialog }: Props): ReactElement {
-  const type = video.label === 'series' ? 'series' : 'collection'
-  const title = video.title[0].value
-  const length = video.children.length
-  const image = video.image
+export function ContainerHero({
```
Comment (csiyang) [inline apps/watch/src/components/VideoContainerPage/ContainerHero/ContainerHero.tsx:17]: Merging main should resolve all the Hero changes

File: apps/watch/src/components/VideoCard/VideoCard.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+const VideoCardStory: ComponentMeta<typeof VideoCard> = {
+  ...watchConfig,
+  component: VideoCard,
+  title: 'Watch/VideoCard'
+}
+
+const Template: ComponentStory<typeof VideoCard> = (args) => {
+  return (
+    <Grid container spacing={2} rowSpacing={8}>
```
[the remark this answers is not in the document]
Comment (tataihono) [reply apps/watch/src/components/VideoCard/VideoCard.stories.tsx:15]: nope look closer there are prop changes in each item.

**Review by csiyang — APPROVED** [review]

**Pull request merged**